R-notebooks/make.R

```diff
@@ -1,7 +1,6 @@
 file_names = list.files(".", pattern="*.Rmd")
 file_names = substr(file_names, 1, nchar(file_names)-4)
 
-file_name = "fb_dashboard"
 for (file_name in file_names) {
   cat(sprintf("Rendering %s ... ", file_name))
```

Comment:
fyi, fb_dashboard took a while to run and broke when I tried this (but I may not have had a clean environment). Doesn't look like you touched it though.

Reply:
I had the same issue

Reply:
I can reproduce this -- will follow up separately since it's not affected by this PR.

Reply:
Tracking in #211

Reply:
Thanks for creating the bug